### PR TITLE
Update messages.json RUS

### PR DESCRIPTION
**Pull request by wvxwxvw**

### CODE REVIEW

File: webextensions/_locales/ru/messages.json [hunk 1]
```diff
@@ -79,9 +79,6 @@
   "bookmarkFolder_label_default": { "message": "$TITLE$ и больше (%YEAR%.%MONTH%.%DATE%)",
     "placeholders": {
       "title": { "content": "$1", "example": "Title" },
```
Comment (piroor) [inline webextensions/_locales/ru/messages.json:81]: The trailing comma needs to be removed. Could you update this PR?

Comment (wvxwxvw) [reply webextensions/_locales/ru/messages.json:81]: I don't know how to update. I will create a new one.

Comment (piroor) [reply webextensions/_locales/ru/messages.json:81]: You'll just commit more changes to [the branch for the pull request](https://github.com/wvxwxvw/treestyletab/tree/patch-2) (you can see that on a link in the line "wvxwxvw wants to merge 1 commit into piroor:trunk from wvxwxvw:patch-2" below the subject of this page). Then the pull request corresponding to the branch will be updated automatically. Please try that when you mistakenly create a pull request including something error.

Comment (wvxwxvw) [reply webextensions/_locales/ru/messages.json:81]: OK. Next time. )

**Pull request closed without merging**